app/controllers/api/v1/photos_controller.rb

```diff
@@ -1,13 +1,29 @@
 class Api::V1::PhotosController < ApplicationController
+  before_action :photo_params
```

Comment:
This doesn't need to be done as a `before_action`. You can instead call the `photo_params` method where it's needed, instead of setting the `@params` instance variable.